### ISSUE TITLE
UI: Update Admin Tools Scans Table

### ISSUE THREAD

**Issue by hawkishpolicy** [cisagov/XFD#344]
# 🐛 Summary #
The Admin Tools Scans table needs to be updated to the MUI Data Grid component to stay inline with the rest of the app.

It will also need proper aria-labels and tab indexes for assistive technology users to meet 508 compliance:

- The “Delete Column” “X” buttons inside the Scans table are not keyboard accessible. There is no mechanism in which keyboard only or assistive technology users can delete a scan because the interactive element is not keyboard accessible using the tab keys or the arrow keys.
This happens because the <svg> is coded within a non-interactive <span> element rather than as a <button>.

- The “Delete Column” “X” buttons inside the Scans table do not have accessible name specified for the buttons. Each “X” button should have a Unique identifier to distinguish it from the other rows in the table. For example: the aria-label for the first row should be “Active port scan of common ports Delete” (assuming that Description is a unique value.)

## Any helpful log output or screenshots ##

![image](https://github.com/cisagov/XFD/assets/123108455/9edb9529-7bb4-4301-b2bf-afddfcfcd0b7)

![image](https://github.com/cisagov/XFD/assets/123108455/73f6167e-6d25-4bea-a8cf-80da4ecbebf1)
